### ISSUE TITLE
Rearrange source list icons

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-client#280]
**Epic:** UX Parity between prototypes & coded client (#261) 

**This Story:** Rearrange source list icons

---
**Details, This Story:**
- Restructure the SourceWidget to show the timestamp, star, delete, and attachment icons in locations according to https://zpl.io/V4OMBzy

**Comment by ninavizz** [comment]
@creviera The gradient that spans the footprint behind the _Source List_, the _Conversation Pane_, and the _Reply pane_, is below. It may be a tad light, ping me when ya get it in. 


```
<?xml version="1.0" encoding="utf-8"?>
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 1224 792" style="enable-background:new 0 0 1224 792;" xml:space="preserve">
<style type="text/css">
	.st0{fill:url(#SVGID_1_);}
</style>
<radialGradient id="SVGID_1_" cx="609.095" cy="326.8184" r="279.7393" gradientTransform="matrix(2.065 0 0 1.9595 -644.4514 -243.4455)" gradientUnits="userSpaceOnUse">
	<stop  offset="0" style="stop-color:#F2F8F4"/>
	<stop  offset="0.4802" style="stop-color:#F2F3FA"/>
	<stop  offset="1" style="stop-color:#F8FCFF"/>
</radialGradient>
<rect x="1.9" y="0.5" class="st0" width="1222.9" height="792.9"/>
</svg>
```

**Comment by creviera** [comment]
I'm splitting this Issue into 3 issues to help provide better scope and smaller PRs:

* Background styling: https://github.com/freedomofpress/securedrop-client/issues/299
* Behavior of icons: https://github.com/freedomofpress/securedrop-client/issues/300
* Restructure the SourceWidget to show the timestamp, star, delete, and attachment icons in the correct locations: **Using this issue**

**Comment by creviera** [comment]
> width scaling only on preview column

I'm splitting Message Previews out from this Issue, we can use this existing issue with an update to account for new design work:

* Message preview: https://github.com/freedomofpress/securedrop-client/issues/135

**Comment by eloquence** [comment]
Resolved via #337, which was reviewed in #330.
